src/metabase/models/pulse.clj

```diff
@@ -225,18 +225,18 @@
 
 (s/defn ^:private cards* :- [HybridPulseCard]
   [notification-or-id]
-  (map (partial models/do-post-select Card)
-       (mdb.query/query
```

Comment:
Another place where the `post-select` is done automatically. Another weird Toucan 1 gotcha removed